data-prepper-plugins/opensearch-source/src/test/java/org/opensearch/dataprepper/plugins/source/opensearch/configuration/AwsAuthenticationConfigurationTest.java

```diff
@@ -1,232 +0,0 @@
-/*
- * Copyright OpenSearch Contributors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package org.opensearch.dataprepper.plugins.source.opensearch.configuration;
-
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Nested;
-import org.junit.jupiter.api.Test;
-import org.mockito.ArgumentCaptor;
-import org.mockito.MockedStatic;
-import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
-import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider;
-import software.amazon.awssdk.awscore.AwsRequestOverrideConfiguration;
-import software.amazon.awssdk.regions.Region;
-import software.amazon.awssdk.services.sts.StsClient;
-import software.amazon.awssdk.services.sts.StsClientBuilder;
-import software.amazon.awssdk.services.sts.model.AssumeRoleRequest;
-
-import java.lang.reflect.Field;
-import java.util.Collections;
-import java.util.Map;
-import java.util.UUID;
-import java.util.function.Consumer;
-
-import static org.hamcrest.CoreMatchers.equalTo;
-import static org.hamcrest.CoreMatchers.instanceOf;
-import static org.hamcrest.CoreMatchers.nullValue;
-import static org.hamcrest.CoreMatchers.sameInstance;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.mockStatic;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.verifyNoMoreInteractions;
-import static org.mockito.Mockito.when;
-
-class AwsAuthenticationConfigurationTest {
-
-    private AwsAuthenticationConfiguration awsAuthenticationOptions;
-
-    private final String TEST_ROLE = "arn:aws:iam::123456789012:role/test-role";
-
-    @BeforeEach
-    void setUp() {
-        awsAuthenticationOptions = new AwsAuthenticationConfiguration();
-    }
-
-    @Test
-    void getAwsRegion_returns_Region_of() throws NoSuchFieldException, IllegalAccessException {
-        final String regionString = UUID.randomUUID().toString();
-        final Region expectedRegionObject = mock(Region.class);
-        reflectivelySetField(awsAuthenticationOptions, "awsRegion", regionString);
-        final Region actualRegion;
-        try(final MockedStatic<Region> regionMockedStatic = mockStatic(Region.class)) {
-            regionMockedStatic.when(() -> Region.of(regionString)).thenReturn(expectedRegionObject);
-            actualRegion = awsAuthenticationOptions.getAwsRegion();
-        }
-        assertThat(actualRegion, equalTo(expectedRegionObject));
-    }
-
-    @Test
-    void getAwsRegion_returns_null_when_region_is_null() throws NoSuchFieldException, IllegalAccessException {
-        reflectivelySetField(awsAuthenticationOptions, "awsRegion", null);
-        assertThat(awsAuthenticationOptions.getAwsRegion(), nullValue());
-    }
```

Comment:
We should keep these tests around and add tests for the getter methods in future CRs.